### DISCUSSION TITLE
Preparation of 0.12.0 release

### DISCUSSION THREAD

**Discussion by hmatuschek** [hmatuschek/qdmr#407] [General]
I am in the process to fix the last communication issues with the BTECH DR-1801 A6. Once that branch is merged into `devel`, the devel-branch will be merged into master and the next version can be released. So, if there are some hot-fixes, you would like to see added to the 0.12.0 release, now is the time to bring them up.

Some new/fixed features with the 0.12.0 release:
- Support for BTECH DR-1801 A6
- Support for Radioddity GD-73
- Proper units for frequencies and intervals. Fixes ugly YAML files.
- Some more fixes to minor issues.

**Comment by allesand** [comment]
@hmatuschek : since #375 is fixed, from my side: go ahead!

**Reply by tpalmatier** [reply]
Was wondering if support for the Radioddity GD-88 was in the works.
